### ISSUE TITLE
tomcat 9 is ok .

### ISSUE THREAD

**Issue by xingzhefeng** [ran-jit/tomcat-cluster-redis-session-manager#5]
just  request.getSession().invalidate(); not work

**Comment by ran-jit** [comment]
Hi @xingzhefeng, I will test the request session invalidate and update you as early as possible from my end. Thanks!

**Comment by xingzhefeng** [comment]
now, I use request.getSession().removeAttribute("logininfo") to logout  There's no problem now. Thank you for your hard work . 
tomcat version is 
```
Server version: Apache Tomcat/9.0.0.M26
Server built:   Aug 2 2017 20:29:05 UTC
Server number:  9.0.0.0
OS Name:        Linux
OS Version:     2.6.32-431.29.2.el6.x86_64
Architecture:   amd64
JVM Version:    1.8.0_111-b14
JVM Vendor:     Oracle Corporation
```

**Comment by ran-jit** [comment]
Thanks @xingzhefeng !!

**Update:** I identified few tomcat-7 compatibility issues and fixed.

The fix is available in 2.0.1 release. (https://github.com/ran-jit/tomcat-cluster-redis-session-manager/releases/tag/2.0.1).
